src/routes/write-exercise/components/__tests__/FeedbackSection.spec.tsx

```diff
@@ -40,9 +46,12 @@ describe('Feedback section', () => {
 
   it('should render incorrect feedback', () => {
```

Comment:
Perhaps you could add a short test with `needsToBeRepeated` set to true?

Reply:
done